packages/template-retail-react-app/app/pages/product-list/index.test.js

```diff
@@ -128,37 +141,44 @@ afterEach(() => {
 afterAll(() => server.close())
 
 test('should render product list page', async () => {
+    window.history.pushState({}, 'ProductList', '/uk/en-GB/category/mens-clothing-jackets')
```

Comment:
We don't need to add it since we're already doing this for every test in `beforeEach`.

```suggestion

```